File: app/src/main/java/com/jermaine/awesomelibraries/repository/callback/OnFetchReposCallback.java
package com.jermaine.awesomelibraries.repository.callback;


import com.jermaine.awesomelibraries.api.response.Repo;

import java.util.List;

public interface OnFetchReposCallback {
    void onFetch(List<Repo> responseList);

    void onError();
}

File: app/src/main/java/com/jermaine/awesomelibraries/repository/callback/OnFetchReposCallback.kt
package com.jermaine.awesomelibraries.repository.callback


import android.arch.lifecycle.LiveData

import com.jermaine.awesomelibraries.datamodel.repo.Repo

interface OnFetchReposCallback {
    fun onSuccess(repoList: LiveData<List<Repo>>)

    fun onError()
}
